src/common/Navigation/styled.js

```diff
@@ -1,49 +1,160 @@
+export const NavigationWrapper = styled.div`
+    background: ${({ theme }) => theme.colors.navigation.background};
     color: ${({ theme }) => theme.colors.navigation.elems};
     min-height: 94px;
+`;
 
-    @media(max-width: ${({ theme }) => theme.breakpoints.small}px) {
-        min-height: 142px;
-    }
+export const NavigationContainer = styled.div`
+    max-width: 1368px;
+    margin: 0 auto;
+    min-height: inherit;
 `;
 
-export const Container = styled.div`
+export const DoubleContainer = styled.div`
     display: flex;
-    flex-wrap:wrap;
-    align-items:center;
-    justify-content: center;
-    margin: 0 auto;
-    flex-basis: 50%;
-    padding: 5px;
+    flex-wrap: wrap;
+    min-height: inherit;
+    padding: 0 10px;
 `;
 
 export const HeaderContainer = styled.div`
     display: flex;
     flex-wrap: nowrap;
     align-items: center;
-    justify-content: space-around;
-    flex-basis: 50%;
+    min-height: inherit;
     margin: 0 auto;
-    padding: 0 10px;
+    min-height: 54px;
+`;
+
+export const Logo = styled.a`
+    display: flex;
+    align-items: center;
+    margin-right: 20px;
+`;
+
+export const VideoIcon = styled.img`
+    color: ${({ theme }) => theme.colors.navigation.elems};
+    width: 34px;
+    margin-right: 12px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.xsmall}px) {
+        width: 26px;
+    }
 
-    @media(max-width: ${({ theme }) => theme.breakpoints.small}px) {
-        flex-basis: 90vw;
+    @media(max-width: ${({ theme }) => theme.breakpoints.xxsmall}px) {
+        width: 14px;
     }
+`;
+
+export const Title = styled.h1`
+    text-decoration: none;
+    color: ${({ theme }) => theme.colors.navigation.elems};
+    font-size: 24px;
+    font-weight: 500;
+    margin: 0;
 
     @media(max-width: ${({ theme }) => theme.breakpoints.xsmall}px) {
-        flex-basis: 70%;
+        font-size: 22px;
+    }
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.xxsmall}px) {
+        font-size: 14px;
     }
 `;
 
-export const SearchContainer = styled(HeaderContainer)`
+export const NavigationLinks = styled.ul`
+    display: flex;
+    list-style: none;
+    padding-left: 0;
+    text-decoration: none;
+    color: white;
+`;
+
+export const NavigationItems = styled.li``;
+
+const activeClassName = "active";
+
+export const StyledNavLink = styled(NavLink).attrs(() => ({ activeClassName}))`
+    text-transform: uppercase;
+    font-weight: 600;
+    font-size: 14px;
+    text-decoration: none;
+    padding: 8px 24px;
+    color: ${({ theme }) => theme.colors.navigation.elems};
+
+    &:hover {
+        border: 1px solid ${({ theme }) => theme.colors.navigation.elems};
+        border-radius: 24px;
+    }
+
+    &.${activeClassName} {
+        border: 1px solid ${({ theme }) => theme.colors.navigation.elems};
+        border-radius: 24px;
+        background-color: ${({ theme }) => theme.colors.navigation.elems};
+        color: ${({ theme }) => theme.colors.navigation.background};
```

Comment:
should be:
color: ${({ theme }) => theme.colors.navigation.elems};

Reply:
I added a color change to the active button so this color matches it.